fpga/bootrom/xip/Makefile

```diff
@@ -35,11 +40,11 @@ hex: $(hex)
 
```

Comment:
I can't comment on the above lines since they aren't in the diff for this PR, but I have one question.
Is the hardcoded target_address coming from an RTL config fragment? If so we should think about how to do this without repeating ourselves.

Reply:
Yea. Im not sure. I need to look into this more... they end up using a `MaskROM` over a `BootROM` in the code which is a bit confusing to me.

Reply:
I believe `MaskROM` implies a specific kind of `ROM` that is reprogrammable with just a single VLSI mask update. I think there is an overload used here sometimes for an FPGA `ROM` that can be updated without the rest of the bit stream being changed.

Besides that definition, I think a `MaskROM` can be a `BootROM` (a `ROM` that contains code to boot, and is where the core resets to), but not all `BootROM`s would be `MaskROM`s and not all `MaskROM`s would be `BootROM`s.

Reply:
So it looks like the `MaskROM` is used to load in a ROM after elaboration using `readmemh`. This differs from the BootROM which bakes the `.img` directly into the module in Chisel.

Reply:
Should be good to just swap `MaskROM` for the `BootROM`